reconcile/gitlab_housekeeping.py

```diff
@@ -19,6 +19,26 @@
 
 QONTRACT_INTEGRATION = 'gitlab-housekeeping'
 
+def clean_pipelines(dry_run, gl, pipeline_timeout=60):
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
This is a constant you share with many other methods. If you really need it, make it global.